central/postgres/pruning_test.go

```diff
@@ -578,6 +579,56 @@ func (s *PostgresPruningSuite) TestRemoveOrphanedPLOPs() {
 	}
 }
 
+func (s *PostgresPruningSuite) TestPruneDiscoveredClusters() {
+	datastore := discoveredClustersDS.GetTestPostgresDataStore(s.T(), s.testDB)
+
+	clusters := []*storage.DiscoveredCluster{
+		// Should not be subject to pruning.
+		{
+			Id:            "cd118b6d-0b2e-5ab1-b1fc-c992d58eda9f",
+			LastUpdatedAt: timestamp.TimeBeforeDays(2),
```

Comment:
Why should this not be pruned if the retention time is 1 day, and this was last updated 2 days ago?